lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -2406,9 +2408,14 @@ fn do_channel_holding_cell_serialize(disconnect: bool, reload_a: bool) {
 			assert_eq!(pending_cs.commitment_signed, cs);
 		} else { panic!(); }
 
-		// There should be no monitor updates as we are still pending awaiting a failed one.
-		check_added_monitors!(nodes[0], 0);
-		check_added_monitors!(nodes[1], 0);
+		if reload_a {
+			// The two pending monitor updates were replayed (but are still pending).
+			check_added_monitors(&nodes[0], 2);
+		} else {
```

Comment:
A little confused what these test changes are doing/how they're related to this PR - are they mainly just checking that in-flight updates get replayed if we shutdown between receiving a revoke_and_ack (--> starting to claim downstream) and updating the monitor?

Reply:
Ah okay!